docs/scanners.md

```diff
@@ -177,6 +177,7 @@ Scanners:
 60001   Example file passive rule
 60100   Example simple active rule
 60101   Example file active rule
+60103   Bypassing 403
```

Comment:
This range was/is being used for examples. I'd suggest using one in `400XX` or `900XX`.